test-suite/test-cases/rdfa1.1/xhtml1/0121.sparql

```diff
@@ -1,4 +1,4 @@
 ASK WHERE {
  <http://example.org/> <http://purl.org/dc/elements/1.1/title> "Test Case 0121" .
- <http://example.org/> <http://purl.org/dc/elements/1.1/contributor> "Shane McCarron" .
+ <http://rdfa.info/test-suite/test-cases/rdfa1.1/xhtml1/0121.xhtml> <http://purl.org/dc/elements/1.1/contributor> "Shane McCarron" .
```

Comment:
Previous version is correct. The empty curie (`[]`) is a safe curie which does not resolve to a URI, so it is effectively ignored, leaving `<http://example.org/>` the current URI. The algorithms describe steps when the **current object resource** is set to a non-null value, and this is a null value.